backend/src/main/scala/com/softwaremill/adopttapir/starter/StarterApi.scala

```diff
@@ -0,0 +1,61 @@
+package com.softwaremill.adopttapir.starter
+
+import cats.data.NonEmptyList
+import cats.effect.IO
+import cats.implicits.catsSyntaxEitherId
+import com.softwaremill.adopttapir.http.{Error_OUT, Http}
+import com.softwaremill.adopttapir.starter.StarterApi.Queries._
+import com.softwaremill.adopttapir.util.ServerEndpoints
+import doobie.util.transactor.Transactor
+import sttp.model.StatusCode
+
+class StarterApi(http: Http, starterService: StarterService, xa: Transactor[IO]) {
+  import http._
+
+  private val starterPath = "starter.zip"
+
+  private val starterEndpoint = baseEndpoint.get
+    .in(starterPath)
+    .in(query[String](tapirVersion))
```

Comment:
maybe modelling this as a json body would give us more flexibility? if we introduce more parameters at some point ...

Reply:
Sure good idea!